src/inferences/junctionTree/create-initial-potentials.ts

```diff
@@ -29,18 +29,40 @@ import {
   objectEqualsByIntersectionKeys,
 } from '../../utils'
 
-const createFactorForClique = (network: INetwork, clique: IClique): string[] =>
+/** * Return a list of factors that care included in the given clique.   Each factor can
```

Comment:
the HUGIN algorithm assigns each factor to one and only one clique so that the product of all the clique potentials is the probability density function over the variables.   The previous implementation added factors to every clique that contained both the node and its parents.